### DISCUSSION TITLE
Confusing versioning of docker images

### DISCUSSION THREAD

**Discussion by carloschida** [ory/kratos#2391] [Ideas]
I recently upgraded from v0.8.0 to v0.9.0.
My thought after reading the release notes (https://github.com/ory/kratos/issues/2319) is that it was ready to be used.

Thus, I was expecting that the only change I needed to make was to my config file and my docker-compose file.
In this last one, I changed from v0.8 to v0.9.0.

After reviewing [DockerHub](https://hub.docker.com/r/oryd/kratos/tags?page=1&name=v0.9), it turned out though that `v0.9.0` point to the same tag as `v0.9` which is the same as `v0.9.0-alpha.3`.

If I remember correctly, `0.9.0-alpha.3` < `0.9` = `0.9.0`. [This semantic version comparison page](https://semvercompare.azurewebsites.net/?version=0.9&version=0.9.0&version=0.9.0-alpha.3) seems to support my memory.

The point I'm trying to make here is that using `v0.x` for a `v0.x.y-alpha.z` is a bit contradicting with SemVer even if all the current releases are alpha. What if we just didn't tag anything as non-alpha on DockerHub?

**Comment by aeneasr** [comment]
Hey there, that's a great point. We should probably tag things explicitly. We introduced this naming convention to help developer stay on e.g. a v0.9.0 branch. I think one problem we are facing if we change this is that anyone on e.g. v0.9 will not receive any new patches, even though they would be used to it from the current way it's working. Do you have an idea how to resolve this?
